src/test/suite/StartIntegration.test.ts

```diff
@@ -118,6 +114,40 @@ suite('Check can deploy default examples', () => {
 
 });
 
+async function cleanDeployedIntegration() {
+	let deployedTreeNode: TreeNode | undefined = await retrieveDeployedTreeNode();
+	if (deployedTreeNode) {
+		await vscode.commands.executeCommand('camelk.integrations.remove', deployedTreeNode);
+		try {
+			await waitUntil(() => {
+				return getCamelKIntegrationsProvider().getTreeNodes().length === 0;
+			}, UNDEPLOY_TIMEOUT);
+		} catch (error) {
+			console.log('Error while trying to undeploy, try to print more information on the running integrations:');
```

Comment:
I might phrase this a little differently. "Error while trying to remove deployed integration:" I think it's obvious that you have additional details, so you don't need to say that you're trying to print them.